pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1311,6 +1311,8 @@ public void deleteOfflineTable(String tableName) {
     if (!_helixAdmin.getResourcesInCluster(_helixClusterName).contains(offlineTableName)) {
```

Comment:
u r right. I thought that is handled at a higher level. Or, perhaps each of these calls are idempotent and run fine. i am not sure. But then we can do that as a separate PR. Not deleting segments is biting us now, so I will check this in